Clover/app/src/main/java/org/floens/chan/core/model/PostLinkable.java

```diff
@@ -88,6 +88,10 @@ public void updateDrawState(@NonNull TextPaint ds) {
         }
     }
 
+    public boolean getSpoilerState() {
```

Comment:
this might be better named as `isSpoilerVisible` since it returns a boolean. `get` kinds makes it seem it will return an enum rather than a boolean

[src](https://stackoverflow.com/a/5322666/615743)

Reply:
Yep, I should be using that convention. State implies more than two states usually.